packages/blade/src/components/Button/BaseButton/StyledBaseButton.web.tsx

```diff
@@ -1,6 +1,6 @@
 import styled from 'styled-components';
 import getStyledBaseButtonStyles from './getStyledBaseButtonStyles';
-import type { StyledBaseButtonProps } from './StyledBaseButton.d';
+import type { StyledBaseButtonProps } from './types';
```

Comment:
I'm just generally avoiding .d.ts files

Reason being: 

1. They are not type-checked
2. They were could interfere with typescript's resolution if we accidentally reexport .web files

Reply:
Sounds good to me, anyway `types` is a clearer name for the module